dependencies.tsv

```diff
@@ -19,7 +19,7 @@ github.com/joyent/gosdc	git	2f11feadd2d9891e92296a1077c3e2e56939547d	2014-05-24T
 github.com/joyent/gosign	git	0da0d5f1342065321c97812b1f4ac0c2b0bab56c	2014-05-24T00:07:34Z
 github.com/juju/ansiterm	git	b99631de12cf04a906c1d4e4ec54fb86eae5863d	2016-09-07T23:45:32Z
 github.com/juju/blobstore	git	06056004b3d7b54bbb7984d830c537bad00fec21	2015-07-29T11:18:58Z
-github.com/juju/bundlechanges	git	cd189848b313df50084de692b05861210022cf81	2017-08-02T17:04:42Z
```

Comment:
any idea why bundlechanges or charm dependency was changed?

Reply:
only the timestamp changed. I'm assuming they didn't use "make rebuild-dependencies.tsv" but edited the file by hand.

Reply:
Usually the reason this happens is because someone updated dependencies.tsv file manually rather than using godeps to write the whole thing.